core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -342,6 +342,7 @@ class ShuffleWriteMetrics private[spark](
 class TaskMetricDistributions private[spark](
     val quantiles: IndexedSeq[Double],
 
+    val duration: IndexedSeq[Double],
```

Comment:
This is detected as a binary-incompatibility by MiMa.

Reply:
@AngersZhuuuu is this OK to 'break' as it's it's private[spark]? we just discovered that it was kind of accidentally excluded from a previous change between 3.0 -> 3.1. If so we can just retain the exclusion, just checking.

Reply:
TBH I am not clear what happened since I am not familiar what is mima doing...How should I check about this?

Reply:
I think this one is fine, but a brief recap: Mima tries to detect changes to public APIs, which could cause binary or source incompatibilities, and we generally never allow those in minor releases. This method signature change triggers Mima, although it's actually 'private'; because it's private to a package, it's technically public in the bytecode, and Mima doesn't know it was private[spark] in Scala. So that's fine, it should be 'excluded' in Mima.

The reason this never failed the tests originally was because this method was already excluded, because it changed from 3.0 to 3.1 too. The way Mima was set up, we were reusing exclusions from 3.0 to 3.1 when evaluating changes from 3.1 to 3.2 as well. That could be a problem, if the change from 3.1 to 3.2 was not intended. When we fixed the Mima config, this method popped up as possibly accidentally excluded, so Dongjoon was double-checking.

I think the ones he found are actually OK, just checking.

Reply:
Thanks for you explanation. Will notice this problem when raise pr.